.github/workflows/release-action.yml

```diff
@@ -120,6 +120,17 @@ jobs:
           git tag $TAG_NAME
           git push origin $TAG_NAME
 
+      - name: Close release issues
```

Comment:
@casey-rapnicki-bixal do we have access in actions to gh cli?

Reply:
confirmed: https://github.com/GSA/px-benefit-finder/blob/cfd4e3baec312d6328c5f1aeb058630da0e9d91a/.github/workflows/stale-branch-maintenance.yml#L58